script.sh

```diff
@@ -16,10 +16,9 @@ then
   HT_SERVICE_NAME=helm
 fi
 
-time_start=0
 $commandToRun
 time_end=$SECONDS
```

Comment:
`time_end` is also now unneccessary. The script sets `time_end` to `$SECONDS` then immediately sets `duration` to `$SECONDS` so `time_end` and `duration` will (almost always) be the same.

Then tracepusher only uses `duration` to `time_end` is never used.

Please remove `time_end` as well.

Reply:
yes, agree with you @agardnerIT, 
now it should be ok, right ?